hugo/assets/scripts/arithmetics/arithmetics.tsx

```diff
@@ -180,7 +180,7 @@ class App extends React.Component<{}, AppState> {
                     <div className="border-solid border border-emeraldLangium bg-emeraldLangiumDarker flex items-center p-3 text-white font-mono ">
                         Preview
                     </div>
-                    <div className="border border-emeraldLangium h-full w-full">
+                    <div className="border border-emeraldLangium h-full w-full overflow-hidden overflow-y-scroll">
```

Comment:
Cleanup to `overflow-x-hidden overflow-y-scroll`, because `overflow-hidden` addresses x and y. Looks weird, even if it is working